src/cpp/session/modules/build/SessionBuildErrors.cpp

```diff
@@ -34,6 +34,7 @@
 #include <session/projects/SessionProjects.hpp>
 
 #define kAnsiEscapeRegex "(?:\033\\[\\d+m)*"
+#define kAnsiUrlRegex "(?:\u001B]8;[^\u0007]*\u0007)*"
```

Comment:
An hyperlink is made of: 

 - `\u001B]8;<params>;<url>;\0007`
 - the text
 - `\u001B]8;;\0007`

The trailing `*` makes this optional